src/components/Button/Button.tsx

```diff
@@ -116,6 +117,7 @@ function Button({
     isDisabled && styles.disabled,
     loading && styles.loading,
     plain && styles.plain,
+    isPlainDestructive && styles.plainDestructive,
```

Comment:
Instead of adding a new className, could you style `.plain.destructive` instead?

Reply:
@danrosenthal Yep, pushed an updated version. Thanks!